arch/x86/kvm/pkvm/mmu.c

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0
+#include <linux/pgtable.h>
+#include <linux/kvm_host.h>
+#include <asm/kvm_pkvm.h>
+#include "early_alloc.h"
+#include "mmu.h"
+#include "bug.h"
+
+static struct pkvm_pgtable pkvm_mmu = {
+	.root = INVALID_PAGE,
+};
+
+static bool mmu_pte_present(void *ptep)
```

Comment:
nit: maybe `mmu_pte_is_present` and `mmu_pte_is_huge` (in line with the field names `pte_is_present` and `pte_is_huge`)? And similarly for ept.

Reply:
Or change the callback as pte_present/pte_huge to make it shorter if it is clear?